backend/src/version/v0_3_2.rs

```diff
@@ -25,6 +49,14 @@ impl VersionT for Version {
         crate::hostname::ensure_hostname_is_set(db, &receipts).await?;
         receipts.id.set(db, generate_id()).await?;
 
+        let mut ui = crate::db::DatabaseModel::new()
```

Comment:
And we move this here since this was the intended behavior, or what was the case during the 0.3.2